### DISCUSSION TITLE
How to reduce margin between page titles and navbar?

### DISCUSSION THREAD

**Discussion by jannes** [hugo-sid/hugo-blog-awesome#162] [Q&A] [answered]
I tried to look through the CSS how to adjust the margin between the "navbar" and "wrapper" divs, but couldn't figure it out. Could someone help me out and point me in the right direction? :)

**Comment by jannes** [comment, marked as answer]
Should have waited a bit longer with my question, I did find the solution myself now:

To adjust margins for the homepage change the `.author` class in `_layout.scss`, to adjust the margins of the posts page and individual posts change the `.header` class in `_post.scss` :)

**Reply by hugo-sid** [reply]
You are right @jannes. I'm glad you were able to find the solution yourself.
